pretests/log_versions/log_versions.py

```diff
@@ -0,0 +1,50 @@
+r"""
+Summary
+-------
+
+Preserve docker versions in state files.
+
+Operational Summary
+-------------------
+
+#. Run 'docker version'; use DockerVersion module to parse output
+#. Run 'rpm -q docker|docker-latest' (whichever one is in use)
```

Comment:
Why limit it just to docker?  Howabout have the test accept a config. option with a list of rpm's to query?  e.g. docker relies on runc and there are other OCI nubbins we're interested in (generally).  Would be handy to have that data recorded also.